src-java/network-topology/network-storm-topology/src/main/java/org/openkilda/wfm/topology/network/storm/bolt/isl/command/IslRulesFailedCommand.java

```diff
@@ -15,17 +15,21 @@
 
 package org.openkilda.wfm.topology.network.storm.bolt.isl.command;
 
-import org.openkilda.wfm.share.model.Endpoint;
-import org.openkilda.wfm.share.model.IslReference;
+import org.openkilda.wfm.topology.network.storm.ICommand;
 import org.openkilda.wfm.topology.network.storm.bolt.isl.IslHandler;
 
-public class IslDefaultRuleTimeoutCommand extends IslCommand {
-    public IslDefaultRuleTimeoutCommand(Endpoint source, Endpoint destination) {
-        super(source, new IslReference(source, destination));
+import java.util.UUID;
+
+public class IslRulesFailedCommand implements ICommand<IslHandler> {
```

Comment:
Better make 2 inheritance branches for IslHandler commands:

```
ICommans<IslHandler> : IslCommandBase => IslCommand => [all old commands]
                                      => IslWorkerCommand => [new commands]
```

Reply:
Added intermediate interface.